app/assets/stylesheets/custom.css.scss

```diff
@@ -10851,7 +10851,10 @@
     border-radius: 10rem;
     padding: 1.5rem 1rem;
   }
-  
+  .form-control-textarea {
+    font-size: 0.8rem;
+    padding: 1.5rem 1rem;
+  }
```

Comment:
@AkshayPashnathSuresh 
these changes are NOT allowed in this PR. 
this PR is meant for `Feature/add change db sqlite pg`. 

In short move all changes other than db change to a separate PR

Reply:
ok, I understood why I happens. 
Its because you created a branch from child branch. 

so from next time, create a new branch from master or change the base to child branch :)